digitalocean/app/app_spec.go

```diff
@@ -260,6 +260,11 @@ func appSpecImageSourceSchema() map[string]*schema.Schema {
 			Optional:    true,
 			Description: "The repository tag. Defaults to latest if not provided.",
```

Comment:
```suggestion
			Description: "The repository tag. Defaults to latest if not provided. Cannot be specified if digest is provided.",
```
Nice changes :raised_hands:  
Maybe add this note to the `tag` as now `digest` is supported?